x/incentives/simulation/operations.go

```diff
@@ -117,7 +115,7 @@ func SimulateMsgCreateGauge(ak stakingTypes.AccountKeeper, bk stakingTypes.BankK
 	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
 		simAccount, _ := simtypes.RandomAcc(r, accs)
 		simCoins := bk.SpendableCoins(ctx, simAccount.Address)
-		if simCoins.Len() <= 0 || simCoins.AmountOf(appparams.BaseCoinUnit).LT(types.CreateGaugeFee) {
+		if simCoins.Len() <= 0 || simCoins.AmountOf(simCoins[0].Denom).LT(types.CreateGaugeFee) {
```

Comment:
nit: suggest `sdk.DefaultBondDenom` since we specifically expect this denom here